### ISSUE TITLE
Nexquito | Nextruder Mosquito hotend

### ISSUE THREAD

**Issue by ByJuix** [gregsaun/prusa_i3_bear_upgrade#139]
Hey there, I plan to modify the Nextruder to allow a Slice Engineering Mosquito hotend to be mounted. I'm sure it'll be a bit of a challenge :D

**TLDR:** 
I would appreciate it if somebody with a MK4 could try to attach some weight to the extruder and see if the first layer calibration succeeds. 

---------
I own a Prusa MK3S+ Bear and would like to keep my Mosquito hotend. I ordered the MK4 upgrade kit, but I think it won't ship within the next two weeks; therefore, it would be good to know if the first layer calibration still succeeds if there is added weight on the extruder assembly. The Mosquito is roughly 10g to 15g heavier than the stock Nextruder heatblock; also there are other printed parts needed, in total it will be around 20g to 30g heavier. 

To make this work, the Mosquito will be mounted on top of the loadcell and the gearbox and motor have to be inverted to stick out the front so that the filament can be fed into the heatbreak, that whole construction then has to be turned so that the nozzle can be aligned with the regular position.

Is it worth the hassle? Probably not, but it would be cool to know if it's possible and maybe @BondtechAB can design a better solution.

_Originally posted by @ByJuix in https://github.com/gregsaun/prusa_i3_bear_upgrade/issues/138#issuecomment-1601633566_

**Comment by ByJuix** [comment]
I created a quick sketch to visualize my idea.

The red part represents the loadcell, the gray part the fan. I'm not certain how to mount the mosquito, but I think I will get some ideas within the next couple of days. [The nozzles are at the exact same location]
![nexquito](https://github.com/gregsaun/prusa_i3_bear_upgrade/assets/43349016/0119b5a1-a984-402c-85cc-59901168daf0)
![nextruder](https://github.com/gregsaun/prusa_i3_bear_upgrade/assets/43349016/92837548-7697-4cd7-bc75-6c3ddc16b735)

**Comment by gregsaun** [comment]
As there is no more activity here and I have no plan to do this I close this issue. Feel free to reopen in case of.
